### PR TITLE
Implemented Fyber adapters

### PR DESCRIPTION
**Pull request by AviGelkop**
Fyber's Marketplace adapter for the Google AdMob mediation framework. Supports Fyber_Marketplace_SDK version 7.4.0 and Google-Mobile-Ads-SDK version 7.49.0
Banners and Interstitial are supported via the old API. Rewarded ads are supported via the new Adapter's API

### CODE REVIEW

File: adapters/Fyber/FyberAdapter/GADMediationAdapterFyber.m [hunk 1]
```diff
@@ -1,81 +1,243 @@
...
+        builder.adRequest = request;
+        [builder addSupportedUnitController:self.viewUnitController];
+    }];
+}
+
+- (void)initInterstitialWithRequest:(IAAdRequest * _Nonnull)request {
+    _MRAIDContentController = [IAMRAIDContentController build:^(id<IAMRAIDContentControllerBuilder>  _Nonnull builder) {}];
+    _videoContentController = [IAVideoContentController build:^(id<IAVideoContentControllerBuilder>  _Nonnull builder) {}];
+
+    _viewUnitController = [IAViewUnitController build:^(id<IAViewUnitControllerBuilder>  _Nonnull builder) {
```
Comment (maninomula) [inline adapters/Fyber/FyberAdapter/GADMediationAdapterFyber.m]: It looks like this is for banners, so unclear why to set this on interstitials. The fullscreenUnitController has both video and MRAID demand.

**Comment by maninomula** [comment]
@AviGelkop Can you please address the one change requested so that we can move forward with the certification?

File: adapters/Fyber/FyberAdapter/GADMediationAdapterFyber.m (hunk 1 above)
Comment (AviGelkop) [reply adapters/Fyber/FyberAdapter/GADMediationAdapterFyber.m]: @maninomula You are right, the viewUnitController is redundant here.

**Comment by maninomula** [comment]
@AviGelkop Thanks for resolving the issue. Can you please resolve the conflicts?

**Comment by AviGelkop** [comment]
@maninomula can you merge now?

**Pull request merged**